tests/e2e/manifests/status-crd.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  # name must match the spec fields below, and be in the form: <plural>.<group>
+  name: operatorstatuses.operatorstatus.openshift.io
+spec:
+  # TODO: remove this version: "v1" field and match version used by installer
+  version: "v1"
+  # group name to use for REST API: /apis/<group>/<version>
+  group: operatorstatus.openshift.io
+  # list of versions supported by this CustomResourceDefinition
+  versions:
+    - name: v1
+      # Each version can be enabled/disabled by Served flag.
+      served: true
+      # One and only one version must be marked as the storage version.
+      storage: true
+  # either Namespaced or Cluster
+  scope: Namespaced
```

Comment:
should not this be installed at "Cluster" scope? All operators, including mao,  will then create OperatorStatus CR using this CRD, no?